Minor changes to new "View Logs" doc (#848)
astronomer/docs: astro/view-logs.md

description: View logs for your data pipelines both locally and on Astro.
---

> Astronomer provides access to both Airflow component and task logs. View logs to troubleshoot your data pipelines and better understand the behavior of your tasks and their execution environment.

Reviewer: ```suggestion
View Airflow component and task logs to troubleshoot your data pipelines and better understand the behavior of your tasks and their execution environment.
```
If we condense this statement, we can imply that Astronomer supports these logs by telling users to check them

5. Click **Instance Details**.
6. Click **Log**.

On Astro, Airflow task logs are stored in the data plane on your cloud. On Amazon Web Services (AWS), they are stored in S3. On Google Cloud Platform (GCP), they are stored in Cloud Storage. Task logs are stored indefinitely for existing Astro clusters. The task log retention policy is not currently configurable.

:::info

Astro has early access support for forwarding Airflow task logs and basic Airflow metrics to [Datadog](https://www.datadoghq.com/). To enable log forwarding, [submit a request to Astronomer Support](astro-support.md).

In your request, make sure to provide your Datadog API key and identify on which Astro cluster(s) you want to enable the integration.

If you're interested in forwarding task logs to other third-party logging tools, also submit a request to Astronomer Support.

:::

## Access Airflow component logs locally

To show logs for your Airflow scheduler, webserver, or triggerer locally, run the following Astro CLI command:

```sh
astro dev logs
```

Once you run this command, the most recent logs for these components appear in your terminal window.

> By default, running `astro dev logs` shows logs for all Airflow components. To see logs only for a specific component, specify any of the following flags in your command:

Reviewer: ```suggestion
By default, running `astro dev logs` shows logs for all Airflow components. To see logs only for a specific component, add any of the following flags to your command:
```
I like the original version better just because I always thought of "specify" as "you have to set a value of your own choosing", whereas here you only have to "add" a pre-configured flag.

- `--scheduler`
- `--webserver`
- `--triggerer`

To continue monitoring logs, run `astro dev logs --follow`. The `--follow` flag ensures that the latest logs continue to appear in your terminal window. For more information about this command, see [CLI Command Reference](cli/astro-dev-logs.md).

Logs for the Airflow webserver, worker, and triggerer are not available for Deployments on Astro.

> ## View Airflow Scheduler Logs

Reviewer: ```suggestion
## View Airflow scheduler logs
```

> You can access the past 24 hours of scheduler logs for any Astro Deployment on the **Scheduler Logs** page of the Cloud UI. Logs are color-coded according to their type. Scheduler logs can help you understand scheduler performance and indicate if a task failed due to an issue with the scheduler. For more information on configuring the scheduler on Astro, see [Scheduler Resources](configure-deployment-resources.md#scheduler-resources).

Reviewer: ```suggestion
You can access the past 24 hours of scheduler logs for any Astro Deployment on the **Scheduler Logs** page of the Cloud UI. Logs are color-coded according to their type. Scheduler logs can help you understand scheduler performance and indicate if a task failed due to an issue with the scheduler. For more information on configuring the scheduler on Astro, see [Scheduler resources](configure-deployment-resources.md#scheduler-resources).
```
